src/generator/pagers.ts

```diff
@@ -5,11 +5,12 @@
 
 import { Session } from '@azure-tools/autorest-extension-base';
 import { camelCase } from '@azure-tools/codegen';
-import { CodeModel } from '@azure-tools/codemodel';
+import { CodeModel, SchemaResponse } from '@azure-tools/codemodel';
 import { values } from '@azure-tools/linq';
 import { PagerInfo } from '../common/helpers';
 import { contentPreamble, sortAscending } from './helpers';
 import { ImportManager } from './imports';
+import { open } from 'fs';
```

Comment:
I dont think this is actuall used, the same thing happened to me when I typed op.

Reply:
Ugh I hate that, will fix.